libpod/events/events.go

```diff
@@ -89,7 +89,13 @@ func (e *Event) ToHumanReadable(truncate bool) string {
 		}
 		humanFormat += ")"
 	case Network:
-		humanFormat = fmt.Sprintf("%s %s %s %s (container=%s, name=%s)", e.Time, e.Type, e.Status, id, id, e.Network)
+		if e.Status == Create || e.Status == Remove {
+			for _, netdriver := range e.Attributes {
```

Comment:
This assumes that there will be only ever one attribute which seems rather fragile and unexpected for the next person adding a new attribute. Please check the correct driver name key only here.

Reply:
Yeah this makes sense... Thanks for pointing it out. I have updated the code.